.env.example

```diff
@@ -0,0 +1,8 @@
+BLOB_ACCOUNT_NAME=
+BLOB_CONTAINER_NAME=
+BLOB_ACCOUNT_KEY=
+BLOB_SUB_ID=
+AML_COMPUTE_CLUSTER_NAME=
+AML_COMPUTE_CLUSTER_MIN_NODES=
+AML_COMPUTE_CLUSTER_MAX_NODES=
+AML_COMPUTE_CLUSTER_SKU=
```

Comment:
Will putting some default value for the CLUSTER_SKU help folks get started?

Reply:
added #366 to track since this is out of scope for the PR